### PR TITLE
Run isort in all files.

### PR DESCRIPTION
**Pull request by Victorsitou**
## Summary

This is an extended PR of #169, I decided to create a separated PR for this.

## Checklist

- [ ] If code changes were made then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `black .`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by onerandomusername** [comment]
BTW, might be worth adding config for isort.

Most importantly: https://pycqa.github.io/isort/docs/configuration/profiles.html#black

File: pyproject.toml [hunk 1]
```diff
@@ -1,10 +1,3 @@
 [tool.black]
 line-length = 100
 target-version = ['py38', 'py39']
-
-[tool.isort]
```
Comment (onerandomusername) [inline pyproject.toml]: This should stay in the configuration, so it doesn't change in the future.

Comment (Victorsitou) [reply pyproject.toml]: I'm doing all the configuration in #169.

**Review by shiftinv — APPROVED** [review]
Scrolled through the entire diff, couldn't find any (obvious) issues. A quick local test also seems to be working fine, so lgtm

**Review by onerandomusername — APPROVED** [review]
rebased on master and ran changes, worked fine

**Pull request merged**